### PR TITLE
Leaderboard Final Touch

### PR DESCRIPTION
**Pull request by algerina**
In this Milestone, I improved the look of the application by adding styles.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @algerina, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
```
Comment (mke2111) [inline README.md:6]: - [x] Add a project name in the readme file.

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 > This is a Leaderboard list app, setup using webpack and ES6 modules.
 
-![screenshot](./screen.JPG)
+![screenshot](./screen2.JPG)
```
Comment (mke2111) [inline README.md:10]: - [x] Add clear instructions to follow to set up on a local computer. I have followed your instructions but I get an error when I run `npm run start`.
Remember to build your app with webpack, also indicate which files should be opened if any.

**Review by mikethreels — APPROVED** [review]
Hi Amel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**